rules/forbidden_actions.tex

```diff
@@ -103,24 +103,26 @@ \subsubsection{Penalty Area During Penalty Kick}
 If a robot enters the relevant penalty area during a penalty kick, except for the goalkeeper (defending team) and one robot of the attacking team, ``Illegal Position'' is called.
 
 \subsection{Forbidden Motion}
-Robots that begin moving their legs or locomote in any fashion before reaching the specified state in the subsections
-will be penalized \textit{in place} on the field for \qty{15}{\second} in \textit{playing}. It's important to note that the penalty duration only counts down during the playing state.
+Robots that begin moving their legs or locomote in any fashion before reaching the specified state in the following subsections
+will be penalized \textit{in place} on the field.
 ``Forbidden Motion'' penalties do not follow the standard removal procedure, and hence do not count towards the incremental penalty count.
-A robot will be moved back to its original position if it has moved significantly before becoming penalized.
+A robot will be moved back to its original position if it has moved significantly before being penalized.
 
 \subsubsection{Motion in Standby}
 \label{sec:motion_in_standby}
 
-Robots that begin to move during \texttt{standby} (\ie before the head referee does the visual signal).
-The head referee will call ``Forbidden Motion in Standby\textless robot\textgreater''.
-Note that responding to any false signal of any fashion will result in this penalty.
+If a robot begins to move during \texttt{standby} (\ie before the head referee does the visual signal), all robots from that team will be penalized until the \textit{ready} state is send.
+This means the Champion Cup will be penalized until the end of the \textit{ready} state, while the Challenge Shield will face penalties until 10 seconds remain in the \textit{ready} state.
```

Comment:
If a robot begins to move during \texttt{standby} (\ie before the head referee does the visual signal), all robots from that team will be penalized for the duration of the delayed \textit{ready} state.
Specifically, for the Champion Cup, this penalty will last up to \qty{\ReadyDelayTimeChampion}{\second}, while for the Challenge Shield, it will extend up to \qty{\ReadyDelayTimeChallenge}{\second}.

Reply:
If a robot begins to move during \texttt{standby} (\ie before the head referee does the visual signal), all robots from that team will be penalized from that moment until the ready signal delay ends.
Speicfically, the penalty will end on the 45th second of ready in CC and on the 35th second in CS.